.github/workflows/zips.yml

```diff
@@ -31,7 +32,10 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-composer-
       - name: Composer Install
-        run: composer install --no-dev --no-interaction --no-progress
+        uses: php-actions/composer@v4
+        with:
+          dev: no
+          php_version: 7.3
```

Comment:
@eugene-manuilov why not use the same `setup-php` action we use for other jobs and keep using the local version of `composer`?

Reply:
Good to go for now, I'm gonna merge. We can address this later if needed 👍

Reply:
@aaemnnosttv I don't see where we use the `setup-php` action. Can you give me a link to it?